plugin/src/test/groovy/com/novoda/test/LogsSubject.groovy

```diff
@@ -44,6 +44,10 @@ class LogsSubject extends Subject<LogsSubject, Logs> {
         check().that(actual().output)
     }
 
+    public void isEmpty() {
```

Comment:
`nit:` this is new but doesn't look like it's used anywhere in the PR

Reply:
Oh, I used it and then replaced it with verifyZeroInteractions